src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubConfiguration/help-usersToIgnore.html

```diff
@@ -0,0 +1,17 @@
+<div>
+    <p>
+        A comma-delimited list of Git usernames to ignore push events from. These events will be handled
+        and logged by the plugin but will not trigger builds. The obvious use-case for this is
+        ignoring push events by a designated Jenkins git user, so that your builds don't trigger
+        more builds.
+    </p>
+    <p>
+        This must <strong>exactly</strong> match their Github username.
+    </p>
+    <p>
```

Comment:
I recommend you to use:
```
<dl>
  <dt>Examples:</dt>
  <dd>jenkinsuser</dd>
  <dd>jenkinsuser,otheruser,anotheruser,yetanotheruser</dd>
</dl>
```

The proposed HTML is wrong from a semantic perspective.